go.mod

```diff
@@ -89,7 +89,7 @@ require (
 	github.com/hashicorp/vault-plugin-secrets-ad v0.8.0
 	github.com/hashicorp/vault-plugin-secrets-alicloud v0.7.0
 	github.com/hashicorp/vault-plugin-secrets-azure v0.8.0
-	github.com/hashicorp/vault-plugin-secrets-gcp v0.8.0
+	github.com/hashicorp/vault-plugin-secrets-gcp v0.8.2
```

Comment:
Hmm.. something about this diff looks off. I would've expected going from `v0.8.1` -> `v0.8.2` instead of `v0.8.0` -> `v0.8.2`. That might explain why there is a change to `role_set.go` in this diff. Might want to rebase this branch from the latest in `release/1.6.x`.